projects/infrastructure-for-perf-tracking.adoc

```diff
@@ -0,0 +1,163 @@
+
+=== Prerequisites
+
+Contributors taking this project needs:
+
+* Basic knowledge about performance testing and tuning
+* Basic knowledge about how to establish a website and tuning JS and PHP code.
+* Have RISC-V physical board (not necessary if you just want a test)
+* Basic knowledge about how to write bash and python scripts.
+
+
+=== Deliverables
+
+* An open-source toolset for building compilers & running benchmarks & show perf results on web.
+* Data reports generated by this tracking platform.
+
+A monthly report to the SIG is a routine deliverables for all projects, which will cover
+- key progress in the past month
+- table showing tasks to be done, in progress, complete and verified
+- test/benchmark results
+- updates to the risk register
+- plans for the next month
+
+=== Milestones
+
+I think it is just a small project in the first place so the milestone is the phase stage itself.
+
+=== Costs
+
+* equipment costs
+  - Can we just get some boards donated by the member companies made them?
+  - An VPS/IaaS for hosting the website. PLCT has existing VPS and can donate for it.
+* license costs
+  - Use free & open benchmark first.
+  - How to donate? Is it possible? (due the LINCENSE of each benchmarks)
+* personnel costs
+  - TBD. Roughly 2 person 1 day per week?
+
+== Risk register
+
+TBD. The PLCT Lab is willing to contribute engineers.
+
+Risks are assessed by the Impact (I) they have on the project from 1 (minor) to project killer (3) and by the Likelihood (L) of the risk occurring from 1 (10% chance) through 10 (100% chance).  The two are multiplied to give an overall Risk Factor (R).  Mitigation must be provided for any risk with I = 3 or R >= 10.
+
+[cols="<4,1,1,1,<4",options="header,pagewidth",]
+|=============================================================================
+| _Risk_  | _I_ | _L_ | _R_ | _Mitigation_
+| No silicon available for testing | 2 | 6 | 12 | Use cycle accurate models
+| Too few engineers committed by members |
+
+3 | 2  | 6 |
+
+RISC-V International to fund contract engineers to do the work
+|=============================================================================
+
+The risk register will be maintained on an ongoing basis.
+
+== Support
+
+* Physical boards are needed.
+  - Especially the boards that can run Linux are welcome.
+* Members can run scripts in their own boards and upload the data.
+* Need commercial toolchains & runtimes run the scripts and send back the performance data to the tracking platform.
+
+
+Table of member organizations and commitments
+
+[cols="<4,<4,1,1,1",options="header,pagewidth",]
+|=============================================================================
+| _Organization_  | _Commitment_ | _Past_ | _2021_ | _2022_
+| PLCT Lab.       | 12 engineer months compiler expertise | X | X |
+| Welcome Inc        | €50k funding |     |     | X
```

Comment:
I think this line is left over from the template. Unless you really do have Welcome Inc. contributing €50k!

Reply:
Oops. I'll submit a new PR to remove this. Thanks :)